lidar_detector/src/lib/keypoint_detection.cpp

```diff
@@ -395,6 +395,12 @@ pcl::PointCloud<pcl::PointXYZ> keypointDetection(pcl::PointCloud<Lidar::PointWit
 	pcl::PointCloud<pcl::PointXYZ> pattern = processCircles(circles_cloud, cloud_without_ground_floor, config.circle_detection);
 	if (config.visualize) { visualize(cloud_calibration_board, pattern, edges_cloud); }
 
+  if (pattern.size() < 4) {
```

Comment:
tabs vs spaces, and I don't want to retab/reformat this entire file. I'll convert these to tabs to match the style.